src/test/java/com/plaid/client/integration/PaymentCreateTest.java

```diff
@@ -74,4 +94,22 @@ public void testStandingOrderCreateSuccess() throws Exception {
     assertNotNull(response.body().getPaymentId());
     assertNotNull(response.body().getStatus());
   }
+
+  @Test
+  public void testSingleImmediatePaymentWithRequestRefundDetailsCreateSuccess() throws Exception {
+    Options options = new Options().withRefundDetails(true);
+    Response<PaymentCreateResponse> response = createSingleImmediatePaymentWithOptions(client(), options);
+    assertSuccessResponse(response);
+    assertNotNull(response.body().getPaymentId());
+    assertNotNull(response.body().getStatus());
+  }
+
+  @Test
+  public void testSingleImmediatePaymentWithBacsCreateSuccess() throws Exception {
+    Options options = new Options().withBacs(new Bacs("1234567890", "000000"));
```

Comment:
Could we also add a test case with iban as well?

Reply:
⚡